test/unittests/parser_expr_test.cpp

```diff
@@ -455,7 +455,7 @@ TEST(parser_expr, call_indirect_table_index)
     const auto code1_bin = i32_const(0) + "1100000b"_bytes;
     const auto [code, pos] = parse_expr(code1_bin, 0, {}, module);
     EXPECT_THAT(code.instructions,
-        ElementsAre(Instr::i32_const, 0, 0, 0, 0, Instr::call_indirect, Instr::end));
+        ElementsAre(Instr::i32_const, 0, 0, 0, 0, Instr::call_indirect, 0, 0, 0, 0, Instr::end));
```

Comment:
Is there really a single test affected by call immediates?

Reply:
It looks like. All intermediate commits should pass all tests. Immediates for calls are rather simple and at that time we were writing more wat2wasm tests.